apps/publications/src/app/dialogs/remove-authors-dialog/remove-authors-dialog.component.html

```diff
@@ -0,0 +1,27 @@
+<h1 class="page-subtitle">{{'DIALOGS.REMOVE_AUTHORS.TITLE' | translate}}</h1>
+<app-alert [alert_type]="'info'">
+  {{'DIALOGS.REMOVE_AUTHORS.MESSAGE' | translate}}
+</app-alert>
+
+<mat-spinner class="ml-auto mr-auto" *ngIf="loading"></mat-spinner>
+<perun-web-apps-authors-list
+  *ngIf="!loading"
+  [authors]="authors"
+  [displayedColumns]="['id', 'name', 'organization', 'email']">
+</perun-web-apps-authors-list>
+
+<div mat-dialog-actions>
+  <button
+    (click)="onCancel()"
+    class="ml-auto"
+    mat-flat-button>
+    {{'DIALOGS.ADD_AUTHORS.CANCEL' | translate}}
+  </button>
+  <button
+    (click)="onRemove()"
+    class="ml-2"
+    color="warn"
+    mat-flat-button>
+    {{'DIALOGS.REMOVE_AUTHORS.REMOVE' | translate}}
+  </button>
+</div>
```

Comment:
You didnt have to create new dialog. You can reuse universal-remove-items-dialog - in this dialog you can only show the names of the authors to the user, but I think that should be enought